content/en/blog/_posts/2024-09-18-cri-o-1.31.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog
+title: "What's new in CRI-O 1.31"
+date: 2024-09-18
+slug: cri-o-1-31
+author: Sascha Grunert
+---
+
+The CRI-O maintainers are happy and proud to announce that CRI-O
+[v1.31.0](https://github.com/cri-o/cri-o/releases/tag/v1.31.0) has been
+released! This brand new version contains a large list of cool new features,
+bug fixes and smaller enhancements. I would like to take the opportunity to
+guide you through CRI-O's latest and greatest enhancements in the field of
+Kubernetes compliant container runtimes.
+
+### New features
+
+The CRI-O community voted to use the OCI runtime
+[crun](https://github.com/containers/crun) as new default in replacement to
+[runc](https://github.com/opencontainers/runc). That's actually not too new,
+because crun has been used as default runtime in the packages and static binary
+bundles for quite a while. The runtime offers an overall better performance and
+lower memory footprint than runc. It's efficiency in terms of faster container
+start times and lower memory usage makes it a more optimized runtime for modern
+workloads, for example when it comes to edge use cases as well as running
+WebAssembly (Wasm) workloads.
+
+CRI-O v1.31 also features support for fine-grained `SupplementalGroups`
+([KEP-3619](https://kep.k8s.io/3619)), which allow to control and track how
+supplemental groups are applied to a container process. If you like to learn
+more about the feature itself, then feel free to read through the [corresponding
+Kubernetes v1.31 blog post](/blog/2024/08/22/fine-grained-supplementalgroups-control).
+
+Beside that, the CRI-O maintainers also added support for the Kubernetes image
+volume source alpha feature ([KEP-4639](https://kep.k8s.io/4639)). This feature
+allows users to utilize OCI images and artifacts as custom volume source and
+mount them into containerized workloads. There is another [Kubernetes v1.31 blog
+post available](/blog/2024/08/16/kubernetes-1-31-image-volume-source) which
+covers more details about the functionality and usage of the feature.
+
+CRI-O now supports [sigstore](https://www.sigstore.dev)
+([cosign](https://github.com/sigstore/cosign)) signature verification for
+policies corresponding to a certain Kubernetes
+[namespace](/docs/concepts/overview/working-with-objects/namespaces). This
+means, that policies in the (default) directory
+`/etc/crio/policies/[NAMESPACE].json` will be validated for each pod of the
+corresponding `NAMESPACE`. This will also happen on container creation, which is
+a huge step forward in enforcing sigstore policies for a dedicated Kubernetes
+namespace in comparison to policies which only apply to the whole cluster
+itself.
+
+Beside the support for bigger Kubernetes features, CRI-O v1.31 ships a bunch of
+cool smaller enhancements, for example:
```

Comment:
```suggestion
other enhancements, for example:
```

Small seems to be short changing these features. Let's not classify them as bigger or smaller.